python/lsst/meas/base/plugins.py

```diff
@@ -475,32 +476,64 @@ def getExecutionOrder(cls):
     def __init__(self, config, name, schema, metadata):
         GenericPlugin.__init__(self, config, name, schema, metadata)
         self.cdMatrix11Key = schema.addField(
-            "%s_CDMatrix_1_1" % name,
+            f"{name}_CDMatrix_1_1",
```

Comment:
Overall I'm not a fan of using "CD matrix" as the name for these (just "matrix" or "jacobian" would be better), as that implies a FITS WCS meaning that's subtly incorrect (related to the fact that just getting the CD matrix of the WCS _doesn't_ do we want).  But feel free to leave it for now if changing it would be disruptive in a way that would expand the scope of the ticket.